### PR TITLE
Fix routing guide link

### PR DESCRIPTION
**Pull request by RobinClowers**
I noticed the link to the guide on nested routes was broken, so I fixed it. Loving remix so far, thanks for open sourcing it!

### CODE REVIEW

File: examples/basic/app/routes/demos/about.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function Index() {
         <p>
           Wait a sec...<em>its children</em>? To understand what we mean by
           this,{" "}
-          <a href="https://remix.run/tutorial/4-nested-routes-params">
+          <a href="https://remix.run/docs/en/v1/guides/routing">
```
Comment (MichaelDeBoey) [inline examples/basic/app/routes/demos/about.tsx:34]:
```suggestion
          <a href="https://remix.run/guides/routing">
```

See #778

File: examples/blog-tutorial/app/routes/demos/about.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default function Index() {
         <p>
           Wait a sec...<em>its children</em>? To understand what we mean by
           this,{" "}
-          <a href="https://remix.run/tutorial/4-nested-routes-params">
+          <a href="https://remix.run/docs/en/v1/guides/routing">
```
Comment (MichaelDeBoey) [inline examples/blog-tutorial/app/routes/demos/about.tsx:34]:
```suggestion
          <a href="https://remix.run/guides/routing">
```

See #778

File: packages/create-remix/templates/_shared_js/app/routes/demos/about.jsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ export default function Index() {
         <p>
           Wait a sec...<em>its children</em>? To understand what we mean by
           this,{" "}
-          <a href="https://remix.run/tutorial/4-nested-routes-params">
+          <a href="https://remix.run/docs/en/v1/guides/routing">
```
Comment (MichaelDeBoey) [inline packages/create-remix/templates/_shared_js/app/routes/demos/about.jsx:33]:
```suggestion
          <a href="https://remix.run/guides/routing">
```

See #778

File: packages/create-remix/templates/_shared_ts/app/routes/demos/about.tsx [hunk 4]
```diff
@@ -31,7 +31,7 @@ export default function Index() {
         <p>
           Wait a sec...<em>its children</em>? To understand what we mean by
           this,{" "}
-          <a href="https://remix.run/tutorial/4-nested-routes-params">
+          <a href="https://remix.run/docs/en/v1/guides/routing">
```
Comment (MichaelDeBoey) [inline packages/create-remix/templates/_shared_ts/app/routes/demos/about.tsx:34]:
```suggestion
          <a href="https://remix.run/guides/routing">
```

See #778

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by RobinClowers** [comment]
@MichaelDeBoey Thanks for the review, I fixed that issue, should be good to merge now.

**Comment by ryanflorence** [comment]
We're actually planning to just rewrite this whole guide soon and the templates have changed. Sorry for getting to this so late and thank you for taking the time to send this!

**Pull request closed without merging**